HLTrigger/Configuration/python/HLT_75e33/modules/hgcalLayerClustersL1Seeded_cfi.py

```diff
@@ -21,37 +29,41 @@
             51.444192, 51.444192, 51.444192, 51.444192, 51.444192,
             69.513118, 87.582044, 87.582044, 87.582044, 87.582044,
             87.582044, 87.214571, 86.888309, 86.92952, 86.92952,
-            86.92952
-        ),
-        deltac = cms.vdouble(1.3, 1.3, 5, 0.0315),
-        deltasi_index_regemfac = cms.int32(3),
-        dependSensor = cms.bool(True),
-        ecut = cms.double(3),
-        fcPerEle = cms.double(0.00016020506),
-        fcPerMip = cms.vdouble(
-            2.06, 3.43, 5.15, 2.06, 3.43,
-            5.15
-        ),
-        kappa = cms.double(9),
-        maxNumberOfThickIndices = cms.uint32(6),
-        noiseMip = cms.PSet(
-            refToPSet_ = cms.string('HGCAL_noise_heback')
-        ),
-        noises = cms.vdouble(
-            2000.0, 2400.0, 2000.0, 2000.0, 2400.0,
-            2000.0
-        ),
-        positionDeltaRho2 = cms.double(1.69),
-        sciThicknessCorrection = cms.double(0.9),
-        thicknessCorrection = cms.vdouble(
-            0.77, 0.77, 0.77, 0.84, 0.84,
-            0.84
-        ),
-        thresholdW0 = cms.vdouble(2.9, 2.9, 2.9),
-        type = cms.string('CLUE'),
-        use2x2 = cms.bool(True),
-        verbosity = cms.untracked.uint32(3)
+            86.92952),
+    thicknessCorrection = cms.vdouble(
+        0.77, 0.77, 0.77, 0.84, 0.84,
+             0.84
+    ),
+    sciThicknessCorrection = cms.double(0.9),
+    deltasi_index_regemfac = cms.int32(3),
+    maxNumberOfThickIndices = cms.uint32(6),
+    fcPerMip = cms.vdouble(        2.06, 3.43, 5.15, 2.06, 3.43,            5.15),
+    fcPerEle = cms.double(0),
+    noises = cms.vdouble( 2000.0, 2400.0, 2000.0, 2000.0, 2400.0,
+             2000.0),
+    noiseMip = cms.PSet(
+      scaleByDose = cms.bool(False),
+      scaleByDoseAlgo = cms.uint32(0),
+      scaleByDoseFactor = cms.double(1),
+      referenceIdark = cms.double(-1),
+      referenceXtalk = cms.double(-1),
+      noise_MIP = cms.double(0.01)
     ),
-    timeClname = cms.string('timeLayerCluster'),
-    timeOffset = cms.double(5)
+    use2x2 = cms.bool(True),
+    type = cms.string('SiCLUE')
+  
+  ),
+  detector = cms.string('all'),
+  recHits = cms.InputTag('HGCalRecHit', 'HGCEERecHits'),
+  timeClname = cms.string('timeLayerCluster'),
+  nHitsTime = cms.uint32(3),
+  mightGet = cms.optional.untracked.vstring
+)
+hgcalLayerClustersHSiL1Seeded = hgcalLayerClustersEEL1Seeded.clone(
+  recHits = cms.InputTag('HGCalRecHit', 'HGCHEFRecHits')
+)
+hgcalLayerClustersHSciL1Seeded = hgcalLayerClustersEEL1Seeded.clone(
+  recHits = cms.InputTag('HGCalRecHit', 'HGCHEBRecHits'),
+  plugin = hgcalLayerClustersEEL1Seeded.plugin.clone(type = cms.string('SciCLUE'))
```

Comment:
I'm confused by the fact that this `hgcalLayerClustersEEL1Seeded`, but I don't see the L1T-seeded inputs (I guess `HGCalRecHit:HGCEERecHits` is not L1T-seeded). What am I missing ?

Reply:
Ciao @missirol I was confused as well at first. After some digging in the past, I saw that the L1T filtering is done downstream on the full collection of the HGCAL Clusters. That's why we build all the clusters here, and we filter them later on.
Maybe @swagata87 @SohamBhattacharya could comment further.

Reply:
Hello 

Before this PR, what we had is this kind of things: 
```
HGCBHInput = cms.InputTag("hltRechitInRegionsHGCAL","HGCHEBRecHits"),
HGCEEInput = cms.InputTag("hltRechitInRegionsHGCAL","HGCEERecHits"),
```
where, `hltRechitInRegionsHGCAL` is regional recHits around L1 objects. See here: https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/HLT_75e33/modules/hltRechitInRegionsHGCAL_cfi.py

Is this feature being changed in this PR?

Reply:
Thanks @swagata87 for clarifying even further.
Definitely not intentionally. We will fix it.

Reply:
> Soham

@missirol 
With the aforementioned fix to the L1seeded inputs, I think the changes should be fine.